roottest/root/io/event/CMakeLists.txt

```diff
@@ -1,12 +1,22 @@
-#-------------------------------------------------------------------------------
-#
-# Placeholder file to translate the tests to the new CTest system. Meanwhile we
-# define a CTest test that calls 'make' in ${CMAKE_CURRENT_SOURCE_DIR}
-#
-#-------------------------------------------------------------------------------
-if(NOT MSVC OR win_broken_tests)
-    ROOTTEST_ADD_TEST(roottest-root-io-event
-                      COMMAND ${ROOT_GMAKE_PROGRAM} cleantest
-                      WORKING_DIR ${CMAKE_CURRENT_SOURCE_DIR}
-                      DEPENDS roottest-scripts-utils)
-endif()
+ROOTTEST_GENERATE_DICTIONARY(libTestIoEvent
+                             Event.h
+                             LINKDEF EventLinkDef.h
+                             SOURCES Event.cxx
+                             NO_ROOTMAP
+                             NO_CXXMODULE
+                             FIXTURES_SETUP root-io-event-libTestIoEvent-fixture)
+
+ROOTTEST_GENERATE_EXECUTABLE(TestIoEvent
+                             MainEvent.cxx
+                             LIBRARIES Core RIO libTestIoEventlibgen
+                             FIXTURES_REQUIRED root-io-event-libTestIoEvent-fixture
+                             FIXTURES_SETUP root-io-event-TestIoEvent-fixture)
+
+ROOTTEST_ADD_TEST(write
+                  COMMAND ./TestIoEvent 10 1 1 1
```

Comment:
As the makefile did, we should check the output.  The goal there was to detect unexpected Warning or Error messages (that did not lead to error exit code).

Reply:
But logs were produce by `echo "write success"`  and `echo "read success"` in the Makefile - not by application itself. Why we want to repeat this logic one-to-one? I guess checking return code is enough.

Reply:
> not by application itself. 

Except in case of failure.  I.e we had:
```
Event ...    2>bigeventTest.read.log       && echo "read success" > bigeventTest.read.log
```
So in case of failure the error output was diffed.

So net effect was to display/give 'easy' access to the error output in case of error.

As long as this is still somewhat the case (i.e. that the output can be seen in case of error) then we are fine.